app/views/notifier/invite.markerb

```diff
@@ -1 +1,8 @@
-<%= t('.message', :invite_url => invite_code_url(@invitation_code), :diasporafoundation_url => 'https://diasporafoundation.org/', :diaspora_id => @inviter.try(:diaspora_handle)) %>
+<%= t('.message',
+  invite_url: invite_code_url(@invitation_code),
+  diasporafoundation_url: 'https://diasporafoundation.org/',
+  user: @inviter.try(:diaspora_handle) == @inviter.try(:name) ?
```

Comment:
So no new "nil" method? It could also be used in https://github.com/diaspora/diaspora/issues/6559#issuecomment-314927738